specs/nonbibrecord.proto

```diff
@@ -2,32 +2,36 @@ syntax = "proto3";
 import "status.proto";
 package adsmsg;
 
-message NonBibSolrRecord {
+message NonBibRecord {
     string bibcode = 1;
-    int32 author_count = 2;
-    bool refereed = 3;
-    repeated string simbad_objects = 4;
-    repeated string grants = 5;
-    repeated string citations = 6;
-    float boost = 7;
-    int32 citation_count = 8;
-    int32 read_count = 9;
-    repeated string readers = 10;
-    repeated int32 downloads = 11;
-    repeated int32 reads = 12;
-    repeated string reference = 13;
-    repeated string ned_objects = 14;
-    // data field keys are link sub types, values are counts    
-    map<string, int32> data = 15;
+    bool refereed = 2;
```

Comment:
the fact that order of the elements changed will make old protobuffs fail (essentially, the whole pipeline will need to be re-deployed)

we can do it now, but for the future - remember to never edits the numbers (they can be removed but the old number must be retired)